File: apps/ios/OpenFrame/Model/Photo.swift
import Foundation

struct PhotoAlbum: Identifiable, Codable, Sendable, Equatable {
    let id: String
    let name: String
    let description: String?
    let coverPhotoPath: String?
    let photoCount: Int
    let source: String?
}

struct Photo: Identifiable, Codable, Sendable, Equatable {
    let id: String
    let albumId: String?
    let filename: String?
    let thumbnailPath: String?
    let mediumPath: String?
    let originalPath: String?
    let width: Int?
    let height: Int?
}

File: apps/android/app/src/main/java/us/openframe/app/domain/model/Photo.kt
package us.openframe.app.domain.model

data class PhotoAlbum(
    val id: String,
    val name: String,
    val description: String?,
    val coverPhotoPath: String?,
    val photoCount: Int,
    val source: String?,
)

data class Photo(
    val id: String,
    val albumId: String?,
    val filename: String?,
    val thumbnailPath: String?,
    val mediumPath: String?,
    val originalPath: String?,
    val width: Int?,
    val height: Int?,
)
